core/src/apps/common/request_pin.py

```
@@ -113,27 +138,63 @@ async def verify_user_pin(
+                ctx,
+                prompt,
+                config.get_pin_rem(),
+                allow_cancel,
+                allow_fingerprint,
+                close_others=close_others,
+                standy_wall_only=standy_wall_only,
+                attach_wall_only=attach_wall_only,
+            )
+            config.ensure_not_wipe_code(pin)
+        except Exception:
+            raise wire.PinCancelled("cancle")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)